graph2tac/loader/predict_server.py

```diff
@@ -760,6 +760,11 @@ def parse_args() -> argparse.Namespace:
                         action='store_true',
                         help="Makes data_server check its inner consistency on each update")
 
+    parser.add_argument('--cpu_thread_count', '--cpu-thread-count',
```

Comment:
Why do we have some (but not all) parameters so that we can choose between underscores and dashes? Isn't it easier to just make all parameters use dashes only? Or is there some technical reason?

Reply:
Historically, Fidel used underscores.  I kept mistyping them, so I added alternate dashed versions.  Also, it can sometimes be confusing since in the python code we have to use the underscore version.  I thought having both is the best compromise.

Reply:
I'll just make all of them both.  (I don't want to make them all dashes, since I would have to change my scripts.)